sap-spring-properties/pom.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.sap.cloud.environment.servicebinding</groupId>
+        <artifactId>java-parent</artifactId>
+        <version>0.10.5</version>
+    </parent>
+    <artifactId>java-sap-spring-properties</artifactId>
```

Comment:
(Major)

Please correct me if I'm wrong, but the term "property" (or "properties") only reflect on the one file format. I think Spring call it "(Application) Configuration" and `.properties` is just one file format. Whereas `.yaml` would be even more prominent, right?

If not, then we should challenge the artifact id, since the name cannot be changed after release.

Reply:
well, it's the official term of the thing: https://docs.spring.io/spring-boot/how-to/properties-and-configuration.html
If people really struggle with this then they will run into much more severe problems not related to this lib at all...